basic_hashtable/b_hashtables.c

```diff
@@ -94,7 +107,17 @@ void hash_table_insert(BasicHashTable *ht, char *key, char *value)
  ****/
 void hash_table_remove(BasicHashTable *ht, char *key)
 {
+  int index = hash(key, ht -> capacity);
 
+  if (ht -> storage[index] == NULL)
+  {
+    printf("ERROR: That key not found");
+    return NULL;
+  }
+  else
+  {
+    destroy_pair(ht -> storage[index]);
```

Comment:
after `destroy_pair``ht->storage[index]` still has junk in it and it needs to be cleared out

`ht->storage[index] = NULL;`